test/e2e/index.js

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+import TestViewer from './TestViewer';
+
+const fixtures = [];
+
+const requireFixtures = require.context('./tests', true, /\.(js|ts|tsx)$/);
```

Comment:
What do you think of this folder name instead?

```suggestion
const requireFixtures = require.context('./fixture', true, /\.(js|ts|tsx)$/);
```

Reply:
I would prefer that, yes. I just copied the `test:regressions` approach but happy to follow-up.